web/assets/js/components/gameboard.ts

```diff
@@ -163,19 +164,23 @@ const Gameboard = class extends Component {
       this.allowedToMove = true;
       this.call('stopTimer', [init], 'gameplayers', 'rival');
 
-      //TODO: chech if this.getMoveNumber() return 0 or 1 for before first move.
-      if (this.getMoveNumber() > 1) {
+      // first call only if user is white
+      if (this.firstMove) {
+        this.firstMove = true;
```

Comment:
Am I tired or is the second statement redundant? (if we're firstMove already, then why reset it)

Reply:
No I'm the one who is tired on this one. Nice catch, like that, it's useless...Should be `false`